packages/ui-billboard/src/Billboard/__tests__/Billboard.test.tsx

```diff
@@ -115,7 +115,7 @@ describe('<Billboard />', async () => {
 
       const subject = await mount(<Billboard onClick={onClick} disabled />)
       const billboard = within(subject.getDOMNode())
-      await billboard.click(null, { clickable: false })
+      await billboard.click(undefined, { clickable: false })
```

Comment:
These were not seen by TS until now. The parameters are optional, so one has to use `undefined` instead of `null`, I've indicated this in the commit message